src/common/InMemoryDbBoards.ts

```diff
@@ -0,0 +1,67 @@
+// @ts-check
+
+import { DBTasks } from 'common/InMemoryDbTasks';
+import { remove } from 'lodash';
+import { IBoard } from 'resources/boards/board.model';
+
+const BoardsData: IBoard[] = [];
+
+// GET ALL BOARDS
+const getAllBoards = (): IBoard[] => {
```

Comment:
you should add throw case as returns value or remove 'if' expression from func body